.pre-commit-config.yaml

```diff
@@ -0,0 +1,10 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
```

Comment:
usually, you'd have more hooks defined, like flake8, mypy etc. we can add it later, but it's better to do it now